ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -11,6 +12,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="$(arg arm_prefix)"/>
```

Comment:
Coming back to this a year later:


We're now making a Husky with 2 UR5 arms on it and I'm running into a similar issue. The issue this time is, how to load 2 rms with different prefixes. is it possible to load separate move groups for RViz? I'm guesssing by namespacing the move_group plugin I should be able to achieve this. @gavanderhoorn